driver/razermouse_driver.c

```diff
@@ -4491,6 +4520,71 @@ static ssize_t razer_attr_read_backlight_led_state(struct device *dev, struct de
     return sprintf(buf, "%d\n", response.arguments[2]);
 }
 
+/**
+ * Write device file "hyperpolling_wireless_dongle_indicator_led_mode"
+ */
+static ssize_t razer_attr_write_hyperpolling_wireless_dongle_indicator_led_mode(struct device *dev, struct device_attribute *attr, const char *buf, size_t count)
+{
+    struct usb_interface *intf = to_usb_interface(dev->parent);
+    struct usb_device *usb_dev = interface_to_usbdev(intf);
+    unsigned char mode = (unsigned char)simple_strtoul(buf, NULL, 10);
+    struct razer_report request = {0};
+    struct razer_report response = {0};
+
+    request = razer_chroma_misc_set_hyperpolling_wireless_dongle_indicator_led_mode(mode);
+
+    razer_send_payload(usb_dev, &request, &response);
+
+    return count;
+}
+
+/**
+ * Write device file "hyperpolling_wireless_dongle_pair"
+ */
+static ssize_t razer_attr_write_hyperpolling_wireless_dongle_pair(struct device *dev, struct device_attribute *attr, const char *buf, size_t count)
+{
+    struct usb_interface *intf = to_usb_interface(dev->parent);
+    struct usb_device *usb_dev = interface_to_usbdev(intf);
+    unsigned int pid = (unsigned int)simple_strtoul(buf, NULL, 16);
+    struct razer_report request = {0};
+    struct razer_report response = {0};
+
+    // Step 1: Put in pairing mode
+    request = razer_chroma_misc_set_hyperpolling_wireless_dongle_pair(0x01, 1);
+
+    request.transaction_id.id = 0x1F;
```

Comment:
remove the newline above (same in step2 please)

Reply:
Done!